### PR TITLE
[Feature] Audio Device List UI Test

### PR DESCRIPTION
**Pull request by phileo99**
## Purpose

* added UI Test cases for audio device list

## Does this introduce a breaking change?

```
[ ] Yes
[ X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* Test the code

```
./gradlew clean cDAT
```

## What to Check
Verify that all UI tests still pass

### CODE REVIEW

**Review by ahammer — DISMISSED** [review]

File: azure-communication-ui/azure-communication-ui-demo-app/src/androidTest/java/com/azure/android/communication/ui/callingcompositedemoapp/CallingCompositeAudioDeviceListTest.kt [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package com.azure.android.communication.ui.callingcompositedemoapp
```
Comment (iaulakh) [inline azure-communication-ui/azure-communication-ui-demo-app/src/androidTest/java/com/azure/android/communication/ui/callingcompositedemoapp/CallingCompositeAudioDeviceListTest.kt:1]: file is missing copyright

File: azure-communication-ui/azure-communication-ui-demo-app/src/androidTest/java/com/azure/android/communication/ui/callingcompositedemoapp/matchers/BottomCellViewHolderMatcher.kt [hunk 2]
```diff
@@ -0,0 +1,44 @@
+package com.azure.android.communication.ui.callingcompositedemoapp.matchers
+
```
Comment (iaulakh) [inline azure-communication-ui/azure-communication-ui-demo-app/src/androidTest/java/com/azure/android/communication/ui/callingcompositedemoapp/matchers/BottomCellViewHolderMatcher.kt:2]: file is missing copyright

File: azure-communication-ui/azure-communication-ui-demo-app/src/androidTest/java/com/azure/android/communication/ui/callingcompositedemoapp/robots/SetupScreenRobot.kt [hunk 3]
```diff
@@ -2,12 +2,63 @@
 // Licensed under the MIT License.
```
Comment (iaulakh) [inline azure-communication-ui/azure-communication-ui-demo-app/src/androidTest/java/com/azure/android/communication/ui/callingcompositedemoapp/robots/SetupScreenRobot.kt:2]: copyright

**Review by iaulakh — CHANGES_REQUESTED** [review]

**Review by ahammer — COMMENTED** [review]
I can approve, once copyright is added

File: azure-communication-ui/azure-communication-ui-demo-app/src/androidTest/java/com/azure/android/communication/ui/callingcompositedemoapp/CallingCompositeAudioDeviceListTest.kt (hunk 1 above)
Comment (phileo99) [reply azure-communication-ui/azure-communication-ui-demo-app/src/androidTest/java/com/azure/android/communication/ui/callingcompositedemoapp/CallingCompositeAudioDeviceListTest.kt:3]: fixed

File: azure-communication-ui/azure-communication-ui-demo-app/src/androidTest/java/com/azure/android/communication/ui/callingcompositedemoapp/matchers/BottomCellViewHolderMatcher.kt (hunk 2 above)
Comment (phileo99) [reply azure-communication-ui/azure-communication-ui-demo-app/src/androidTest/java/com/azure/android/communication/ui/callingcompositedemoapp/matchers/BottomCellViewHolderMatcher.kt:4]: fixed

File: azure-communication-ui/azure-communication-ui-demo-app/src/androidTest/java/com/azure/android/communication/ui/callingcompositedemoapp/robots/SetupScreenRobot.kt (hunk 3 above)
Comment (phileo99) [reply azure-communication-ui/azure-communication-ui-demo-app/src/androidTest/java/com/azure/android/communication/ui/callingcompositedemoapp/robots/SetupScreenRobot.kt:2]: fixed

**Comment by phileo99** [comment]
@iaulakh @ahammer all issues resolved and all checks are green

**Review by iaulakh — APPROVED** [review]

**Review by ShaunaSong — APPROVED** [review]

**Review by ahammer — APPROVED** [review]

**Review by lpeixin — APPROVED** [review]

**Pull request merged**